events-scene.js

```diff
@@ -0,0 +1,404 @@
+/**
+ * The `CONFIG` object contains a scenes property which is an array of scene objects. 
+ * Each scene object consists of two properties: `conditions` and `action`. The `conditions` property 
+ * defines the conditions under which the scene should be triggered, and the `action` 
+ * property defines the function to be executed when the conditions are met. 
+ * 
+ * The `conditions` are defined as key-value pairs.
+ * These keys correspond to specific data values received with the event. 
+ * The values associated with these keys can be either a direct value, an object specifying a comparison, or a function that
+ * must return boolean value.
+ * 
+ * The `action` property defines a function that receives event's data as an input. You can write custom code within this function to 
+ * perform specific actions.
+ */
+
+/****************** START CHANGE ******************/
+let CONFIG = {
```

Comment:
This is a great and extensive example. Let's split this into two, so we can provide a pair of BLE events and scenes and general event handlers.

Reply:
https://github.com/ALLTERCO/shelly-script-examples/pull/45/commits/33f315dbb9ade5f6f5d4e7039766d80b2ae69572